### PR TITLE
[zh] sync docs/tasks/debug

### PR DESCRIPTION
**Pull request by howieyuen**
#### previous
`docs/tasks/debug/debug-application-cluster` 

#### current
`docs/tasks/debug/debug-application`
`docs/tasks/debug/debug-cluster`

#### detail
| from | to |
| --- | --- |
| debug-application-cluster/troubleshooting.md | debug/_index.md |
| debug-application-cluster/debug-init-containers.md | debug/debug-application/debug-init-containers.md |
| debug-application-cluster/debug-application.md | debug/debug-application/debug-pods.md |
| debug-application-cluster/debug-running-pod.md | debug/debug-application/debug-running-pod.md |
| debug-application-cluster/debug-application-introspection.md | debug/debug-application/debug-running-pod.md |
| debug-application-cluster/debug-statefulset.md | debug/debug-application/debug-statefulset.md |
| debug-application-cluster/determine-reason-pod-failure.md | debug/debug-application/determine-reason-pod-failure.md |
| debug-application-cluster/debug-cluster.md | debug/debug-cluster/_index.md |

### CODE REVIEW

File: content/zh/docs/tasks/debug/debug-application/debug-pods.md [hunk 1]
```diff
@@ -34,9 +40,9 @@ your Service?
 -->
 ## 诊断问题   {#diagnosing-the-problem}
 
-故障排查的第一步是先给问题分类。问题是什么？是关于 Pods、Replication Controller 还是 Service？
+故障排查的第一步是先给问题分类。问题是什么？是关于 Pod、Replication Controller 还是 Service？
 
-* [调试 Pods](#debugging-pods)
+* [调试 Pod](#debugging-pods)
 * [调试副本控制器](#debugging-replication-controllers)
 * [调试服务](#debugging-services)
```
Comment (JarHMJ) [inline content/zh/docs/tasks/debug/debug-application/debug-pods.md:47]: replication，service 专有名词不翻译会不会比较好？

Comment (howieyuen) [reply content/zh/docs/tasks/debug/debug-application/debug-pods.md:47]: > replication，service 专有名词不翻译会不会比较好？

我赞同，这是从debug-application-cluster目录搬过来的

**Comment by howieyuen** [comment]
cc @tengqm

**Pull request merged**